frontend/javascripts/oxalis/model/sagas/min_cut_saga.ts

```diff
@@ -176,7 +176,10 @@ function removeOutgoingEdge(edgeBuffer: Uint16Array, idx: number, neighborIdx: n
 
 export function isBoundingBoxUsableForMinCut(boundingBoxObj: BoundingBoxType, nodes: Array<Node>) {
   const bbox = new BoundingBox(boundingBoxObj);
-  return bbox.containsPoint(nodes[0].position) && bbox.containsPoint(nodes[1].position);
+  return (
+    bbox.containsPoint(nodes[0].untransformedPosition) &&
+    bbox.containsPoint(nodes[1].untransformedPosition)
```

Comment:
In theory if a bounding box is involved, the transformed positions should be used, right? However, since volume annotation is not allowed if the skeleton is transformed, this should be ok. Still, I see this as a potential future bug source :thinking: Should this maybe be asserted that the skeleton layer is indeed not transformed (for example, once in the beginning when the tool saga is triggered)?

Reply:
Good point! However, I'll remove the min_cut_saga in the next couple of days, anyway. See https://scm.slack.com/archives/C5AKLAV0B/p1706087477274379.